source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
+Be open to suggestions on how your ticket could be improved for someone picking it up.
+
+**Process:**
+
+- **Selection:**
+Before refinement, the PM picks out the stories for the team to discuss.
+- **Discussion:**
+Dive into the details of each story, hash out the acceptance criteria,
+and call out any dependencies or risks.
+- **Estimation:**
+Estimate based on effort and risk using a [planning-poker scoring system](https://en.wikipedia.org/wiki/Planning_poker#Process), allowing the team to have a clear sense of the size and scope of each ticket.
+- **Decompose:**
+Chop up any stories that are too big to handle in a sprint into smaller bits.
+- **Define 'Done':**
+Ensure each story has a clear _Definition of Done_ (DoD) to know exactly when it is crossed the finish line.
+
+**Output:**
+
+- **Refined backlog:**
+Stories that are fleshed out, estimated, and ready for the PM to prioritize
+- **Scored stories:**
+Each story gets a score so we can try to estimate how many stories we should take on based on our prior rate of work.
+- **Approachable tickets:**
+All tickets that go through refinement should have all the info required for any member of the team to pick up.
+
+#### Audience
+
+Data Catalogue Team
+
+### Peer Review
+
+- In the middle of each sprint we have a thirty-minute peer review session
+that provides an opportunity to receive feedback on the current state
+and future plans for the service.
+- [TODO: peer review content]
+
+#### Audience
+
+All of Data Platform and wider audience
+
+### Sprint Retrospective
+
+At the end of the sprint, we:
+
+- Reflect on the sprint’s process, celebrating successes and identifying opportunities for improvement
+- Review the process, not the stories
+- Define how we could improve
+- Share kudos and feedback
+
+#### Audience
+
+Data Catalogue Team
+
+### Sprint Planning
+
+**Purpose:**
+
+Sprint Planning is where we figure out what work we want to commit to for the next sprint. It's where we:
+
+- Discuss and align on the goals for the upcoming sprint
+- Select stories from the refined backlog, ensuring a shared understanding of scope and objectives
+- Define the sprint goal
+- Define the sprint backlog
+
+**Process:**
+
+- **Discussion:** Discuss the stories in the refined backlog, ensuring a shared understanding of scope and objectives, including any work carried from last sprint
+- **Highlighting:** Highlight stories and spikes that are significant to delivery of the sprint goal
+- **Capacity Estimation:** Estimate the capacity of the team for the sprint, and assess whether the tickets brought in match that
+- **Sprint Backlog:** Select the stories that will be brought into the sprint, and assign them to team members where appropriate
+
+**Output:**
+
+- Ensure everyone has stories that they feel they can work on
+- Ensure everyone has a shared understanding of the tasks to deliver this sprint
+
+#### Audience
+
+Data Catalogue Team
+
+## GitHub Practices
```

Comment:
What do you think about pulling this out into it's own page, and maybe having a discussion to review it?

There was a discussion recently on X-Gov slack about this kind of thing, and a team from HMCTS shared a nice example https://hmcts.github.io/cloud-native-platform/standards/pull-requests.html

Reply:
[You and](https://github.com/ministryofjustice/data-catalogue-runbooks/pull/11#discussion_r1542640359) @jemnery [both agree](https://github.com/ministryofjustice/data-catalogue-runbooks/pull/11#discussion_r1542640359). Looks to be worth pulling out into a contributing doc